src/rp2_common/pico_lwip/CMakeLists.txt

```diff
@@ -149,6 +149,7 @@ if (EXISTS ${PICO_LWIP_PATH}/${LWIP_TEST_PATH})
             ${PICO_LWIP_PATH}/src/netif/ppp/polarssl/sha1.c
             )
 
+    pico_add_library(pico_lwip_apps)
```

Comment:
Where did pico_lwip_apps come from?

Reply:
think you are looking at the wrong diffs?